general/press-and-publications.rst

```diff
@@ -0,0 +1,111 @@
+
+- `<https://datatracker.ietf.org/meeting/103/materials/slides-103-gaia-
+  openwisp-a-hackable-network-management-system-for-the-21st-centry-00>`_
+
+django-freeradius by Fiorella De Luca at PyCon Italy:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- video: https://www.youtube.com/watch?v=Yapdso_6EGA
+- abstract: https://www.pycon.it/conference/talks/django-freeradius
+
+Opening Proprietary Networks with OpenWISP
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+*lightning talk at DjangoCon Europe 2017*:
+
+- slides: `<https://www.slideshare.net/FedericoCapoano/opening-propietary-
+  networks-with-openwisp>`_
+
+Do you really need to fork OpenWRT? *at OpenWRT Summit 2015 in Dublin*:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- video: https://www.youtube.com/watch?v=2uioGZuITbA
+
+OpenWISP e Progetti WiFi Nazionali
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+*Interview for GARRTV by Davide Guerri (in Italian)*:
+
+- video: https://www.youtube.com/watch?v=4AE7XSTPCT0
+
+OpenWISP GARR Conference 2011
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+*presented by Davide Guerri (in Italian)*:
+
+- video: https://www.youtube.com/watch?v=4mxiupJNPKo
+
+Blog Posts:
+-----------
+
+- https://medium.com/@williamchoudhury/openwisp-blog-post-ac6987d6b734
+- `<https://blog.p2pfoundation.net/how-bottom-up-broadband-will-overcome-
+  the-last-mile-problem/2013/07/23>`_
+- `<http://nemesisdesign.net/blog/coding/netjsonconfig-convert-netjson-to-
+  openwrt-uci/>`_
+- `<http://nemesisdesign.net/blog/coding/automate-openwrt-lede-firmware
+  -generation-ansible/>`_
+- http://nemesisdesign.net/blog/coding/django-x509-pki-pem/
+- `<http://nemesisdesign.net/blog/coding/network-topology-visualizer-django
+  -netjsongraph/>`_
+- https://uniamo.uniurb.it/openwisp/ (in Italian)
+- https://uniamo.uniurb.it/google-summer-of-code-2018/ (in Italian)
+- https://uniamo.uniurb.it/uniurb-google-summer-of-code-openwisp2/
+  (in Italian)
+- Post by the Metrpolitan City of Rome (in Italian): 
+  `<http://www.cittametropolitanaroma.gov.it/homepage/elenco-siti-tematici
+  /wifimetropolitano/openwisp-la-soluzione-open-source-la-diffusione-
+  servizi-wifi/>`_
+
+Research and publications:
+--------------------------
+
+- `<https://github.com/openwisp/openwisp2-docs/blob/master/docs/netCommons
+  .pdf>`_
+- `<https://github.com/openwisp/openwisp2-docs/blob/master/docs/arxiv.org
+  .pdf>`_
+- `<https://github.com/openwisp/openwisp2-docs/blob/master/docs/
+  JustelPizarro_2013.pdf>`_ (in Spanish)
+- `<https://github.com/openwisp/openwisp2-docs/blob/master/docs/netCommons
+  .pdf>`_ (only one mention but worth having it on the list)
+- `<https://github.com/openwisp/openwisp2-docs/blob/master/docs/upcommons.
+  upc.edu.pdf>`_ (in Catalan)
+
+Logos and Graphic material
+--------------------------
+
+OpenWISP Logo with a Black Foreground:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+`<https://github.com/openwisp/openwisp2-docs/blob/master/images/openwisp.
```

Comment:
you're still linking the github file

Reply:
I copied the files to this repo as you told me. I'm just linking it to the files here :disappointed:

Reply:
When and if my PR would've been merged, it would link these resources correctly... if you want me to do it some other way, let me know...

Reply:
upload the images into this repo